src/pytorch_lightning/utilities/imports.py

```diff
@@ -13,20 +13,11 @@
 # limitations under the License.
 """General utilities."""
 import operator
-import platform
-import sys
 
 import torch
 from lightning_utilities.core.imports import compare_version, module_available, package_available, RequirementCache
 
-_IS_WINDOWS = platform.system() == "Windows"
-_PYTHON_GREATER_EQUAL_3_8_0 = (sys.version_info.major, sys.version_info.minor) >= (3, 8)
-_PYTHON_GREATER_EQUAL_3_10_0 = (sys.version_info.major, sys.version_info.minor) >= (3, 10)
 _TORCH_LESSER_EQUAL_1_10_2 = compare_version("torch", operator.le, "1.10.2")
```

Comment:
why this stay and others are moved?

Reply:
Lite does not define this one. The others are, so they were duplicated